package.json

```diff
@@ -1,37 +1,36 @@
 {
-  "author": "Allan Ebdrup",
-  "name": "nodeerrors",
+  "author": "Debitoor",
+  "name": "@debitoor/nodeerrors",
```

Comment:
I would not do this - it will make you publish an all new package under this scope. In such case, you should use version 1 as the previous versions will _not_ exist for `@debitoor/nodeerrors` - I'd recommend to keep it as is.

Reply:
`nodeerrors` in npm registry is not owned by us, so I guess we won't be able to publish it with this name. 
`@debitoor/nodeerrors` with version 1 sounds better then?

Reply:
@bifrost is a collaborator on https://www.npmjs.com/package/nodeerrors - you can ask him to add you 😄